apps/nextjs-website/src/components/atoms/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment } from 'react';
+import Button from '@mui/material/Button';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Link from 'next/link';
+import { ArrowDropDown } from '@mui/icons-material';
+import { ButtonNaked } from '@pagopa/mui-italia';
+
+type DropdownProps = {
+  label: string;
+  items: { href: string; label: string }[];
+};
+
+const Dropdown = ({ label, items }: DropdownProps) => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  return (
+    <Fragment>
+      <Button
+        aria-controls={open ? `${label}-menu` : undefined}
+        aria-expanded={open ? 'true' : undefined}
+        variant='naked'
+        disableElevation
+        onClick={handleClick}
+        endIcon={<ArrowDropDown />}
```

Comment:
```suggestion
        endIcon={open ? <ArrowDropUp /> : <ArrowDropDown />}
```
from the Figma, if the menu is open, the arrow is upside down.

Note that if you accept this suggestion, you have to take care of the import